manifests/Arion-Air Supperiority.yaml

```
@@ -0,0 +1,20 @@
+name: Arion - Air Superiority
+authors: Spectre907 (With help from 1010todd, yuyuko_saigyouji. and gefulltetaubenbrust)
+homepage: https://github.com/Spectre907YT/Arion-Air_Superiority
+license: CC-BY-SA-4.0
```

Oh, I didn't realize this earlier: CC-BY-SA-4.0 is one-way compatible with GPL3+; so you can have CC-BY-SA-4.0 assets in a GPL3+ plugin, but not the other way round. Part of your plugin [is GPL3+](https://github.com/Spectre907YT/Arion-Air_Superiority/blob/873c60a668feb0cd102a38c9771986b1e6cb0816/copyright.txt#L6), so you can't use CC-BY-SA-4.0 here.

You'll probably want to change this line to
```suggestion
license: GPL-3.0-or-later
```